client/modules/IDE/components/AccessibleOutput.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { PropTypes } from 'react';
+import GridOutput from '../components/GridOutput';
+import TextOutput from '../components/TextOutput';
+
+class AccessibleOutput extends React.Component {
+  componentDidMount() {
+    this.AccessibleOutputModal.focus();
+  }
+  componentDidUpdate(prevProps) {
+    // if the user explicitly clicks on the play button, want to refocus on the text output
+    if (this.props.isPlaying && this.props.previewIsRefreshing) {
+      this.AccessibleOutputModal.focus();
+    }
+  }
+  render() {
+    return (
+      <section
+        className="accessible-output"
+        id="canvas-sub"
+        ref={(element) => { this.AccessibleOutputModal = element; }}
```

Comment:
very minimal fix--could `this.AccessibleOutputModal` be `this.accessibleOutputModal`? I like to reserve uppercase variables for classes.